doc.html

```diff
@@ -109,10 +123,10 @@ <h2>6 Service Template API Overview</h2>
 +--------------------------------------------+
 | logical operation name                     |
 +-------------+------------------------------+
-|      Inputs | &lt;URL parameters or           |
-|             | request body attributes&gt;     |
+|      Inputs | &lt;URL parameters or        |
```

Comment:
The "bad alignment" of the last "|" is just in the editor. When rendered in html, &amp;lt; and &amp;gt; will become < and > respectively. Please put the spaces in all relevant places